### PR TITLE
Fix r- and f-triple-quoted string

### LINKED ISSUE #59: Problem with multi-line script strings
```
rule top:
    shell:
        r"""
        Multi_line
        """
```

becomes

```
rule top:
    shell:
        r"""
                Multi_line
                """
```

becomes

```
rule top:
    shell:
        r"""
                        Multi_line
                        """
```

etc.

### PR DESCRIPTION
**Pull request by bricoletc**
* Fix #59 + add unit test of the problematic example

* Place version in only one place: the pyproject toml (3fe1db1) Then we know we only touch that to update version.

* Shorten README: see commit message of fddf980 
 -->Did this because I like our tool, which has a simple use case and design, to be simple and quick to read through

* Add myself in the LICENCE: muahaha!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,7 +77,7 @@ singularity exec "$URI" snakefmt --help
 The above will use the latest version. If you want to specify a version then use a [tag][dockerhub] like so.
 
 ```sh
-VERSION="0.1.3"
+VERSION="x.x.x"
```
Comment (mbhall88) [inline README.md:80]: keeping this with a valid version number is much clearer

File: README.md [hunk 2]
```diff
@@ -255,91 +252,6 @@ Options:
...
--       input: "foo.txt"
-+     input:
-+         "foo.txt",
-
-[INFO] All done 🎉
-```
-
-If multiple files are specified, a diff for each file is written to stdout, separated by
-`=====> Diff for <filepath> <=====`.
-
```
Comment (mbhall88) [inline README.md:342]: I disagree with deleting this

File: README.md [hunk 3]
```diff
@@ -371,24 +283,6 @@ In this example we increase the `--line-length` value and also include python (`
...
-* Invalid parameters
-* Invalid python code
-
-But `snakefmt` not complaining does not guarantee your file is entirely error-free.
-
-### Formatting
-
-Python code is `black`ed.
-
-Snakemake-specific syntax is formatted following the same principles: see [PEP8][PEP8].
```
Comment (mbhall88) [inline README.md:391]: Moving this text to another file creates more complexity than it solves from a docs point of view. If you are determined to remove this from the README, at least add a link to the new document.

**Review by mbhall88 — CHANGES_REQUESTED** [review]

**Comment by mbhall88** [comment]
@bricoletc can you please also add the changes to the unreleased section in the changelog?

File: README.md (hunk 2 above)
Comment (bricoletc) [reply README.md:342]: why? as i've said, the options already appear in the README in the Full Usage section, and appear at CLI when running `--help`. and even just the keyword itself,  check/diff, is pretty self-explanatory.

File: README.md (hunk 3 above)
Comment (bricoletc) [reply README.md:391]: Happy deleting this then

File: README.md (hunk 1 above)
Comment (bricoletc) [reply README.md:80]: but at least it forces the user to specify which version they want to use. its obvious to me you replace that by your version of interest. we could add a `poetry version | grep -oE "([0-9]\.)+[0-]"` for latest version instead

Comment (mbhall88) [reply README.md:80]: nm

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+## [0.1.4]
+
+- Add parsing support for format and raw (f/r) triple-quoted strings.
```
Comment (mbhall88) [inline CHANGELOG.md:12]:
```suggestion
### Fixed

- Add parsing support for format and raw (f/r) triple-quoted strings [#55][55].
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+## [0.1.4]
```
Comment (mbhall88) [inline CHANGELOG.md:10]:
```suggestion
```

**Review by mbhall88 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (mbhall88) [reply README.md:342]: nm

**Pull request merged**